packaging/building-a-package/en.md

```diff
@@ -61,9 +61,8 @@ Your folder should look something similar to this:
 ```
 | common/
 | your-package/
-| - Makefile
+| Makefile
```

Comment:
The previous bit was intentional, it was meant to illustrate that there would be a Makefile in your-package/